package.json

```diff
@@ -15,6 +15,7 @@
   },
   "devDependencies": {
     "@alrra/travis-scripts": "^3.0.1",
+    "@hint/artwork": "^2.2.0",
```

Comment:
Is it `devDependency` ok?

Reply:
Yes, we are copying them on `postinstall`, so during development. We have them "locally" after that.